lib/galaxy/jobs/runners/slurm.py

```diff
@@ -111,6 +111,7 @@ def _get_slurm_state():
                     check_memory_limit_msg = self.__check_memory_limit( ajs.error_file )
                     if check_memory_limit_msg:
                         log.info( '(%s/%s) Job hit memory limit', ajs.job_wrapper.get_id_tag(), ajs.job_id )
+                        # this fail message is odd since check_memory_limit_msg is True or False
```

Comment:
I believe it's a non-empty string or False

Reply:
You are right. I removed the comment and added documentation to __check_memory_limit.